modules/account-lib/test/unit/coin/eth2/keyPair.ts

```diff
@@ -89,75 +115,185 @@ describe('Eth2 Key Pair', () => {
       );
     });
 
-    it('from an invalid private key', () => {
-      const shorterPrv = '82A34E';
-      const longerPrv = prv + '1';
-      should.throws(
-        () => new KeyPair().recordKeysFromPrivateKey(shorterPrv),
-        (e) => e.message === testData.errorMessageInvalidPrivateKey,
-      );
-      should.throws(
-        () => new KeyPair().recordKeysFromPrivateKey(longerPrv),
-        (e) => e.message === testData.errorMessageInvalidPrivateKey,
-      );
+    it('from a private key', () => {
       should.throws(
-        () => new KeyPair().recordKeysFromPrivateKey(prv + pub),
+        () => new KeyPair().recordKeysFromPrivateKey(prv),
         (e) => e.message === testData.errorMessageInvalidPrivateKey,
       );
     });
   });
 
   describe('should reject key aggregation', () => {
     it('from a single invalid pubkey string', () => {
-      const pub = Buffer.from('abc');
+      const pub = '123';
       should.throws(() => KeyPair.aggregatePubkeys([pub]));
     });
 
     it('from a single invalid pubkey buffer', () => {
-      const pub = Uint8Array.from(Buffer.from('abc', 'hex'));
+      const pub = '0x' + Buffer.from('abc').toString('hex');
       should.throws(() => KeyPair.aggregatePubkeys([pub]));
     });
 
     it('from a set of invalid pubkeys', () => {
-      const pub1 = Uint8Array.from(Buffer.from('abd', 'hex'));
+      const pub1 = '0x' + Buffer.from('abc').toString('hex');
 
       const keyPair2 = new KeyPair();
-      const pub2 = Uint8Array.from(Buffer.from(keyPair2.getKeys().pub.slice(2), 'hex'));
+      const pub2 = keyPair2.getKeys().publicShare;
```

Comment:
out of curiousity why did we modify pub to be publicShare now? is there something wrong with using pub as a field?

Reply:
it is because we are using the BlsKeys type we defined in iface. There, we declared the property as publicShare instead of pub. We can rename it if that is confusing.

Reply:
That's ok thanks for clarification